runtime/drive_handler.go

```diff
@@ -124,6 +124,39 @@ func (h *StubDriveHandler) Reserve(
 	), nil
 }
 
+// Release unmounts stub drive of just deleted container
```

Comment:
I'm thinking we may also need to update `Reserve` to check if `id` is already in `h.usedDrives` and return an error if so. If that situation happened, it would potentially mean we entered some inconsistent state where a task with the given `id` was deleted according to containerd but the backing drive failed to get `Release`'d